src/v/cluster/partition_manager.cc

```diff
@@ -11,6 +11,7 @@
 
```

Comment:
```
application: do not shutdown internal partitions
During redpanda shutdown sequence we stop all partitions to prevent
kafka requests from waiting before shutting down the application.
Internal topics currently raft0 are managed on their own and
they do not have to be shut down early. Shutting down raft0 have to be
performed after we successfully stop all dependent services.
```

what is the issue that this is solving?